week1/task3/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿int[] arr = new[] { 271, -3, 1, 14, -100, 13, 2, 1, -8, -59, -1852, 41, 5 };
+int[] result = GetResult(arr);
+Console.WriteLine($"Sum of negative numbers: {result[0]}. Multiplication of positive numbers: {result[1]}");
+
+int[] GetResult(int[] input)
+{
+    int multiplicationResult = 1;
+    int sumofNegativeNumbers = 0;
+
+    foreach (int number in input)
+    {
+        if (number >= 0)
```

Comment:
here if the number is 0 then the multiplication result will also be 0, the check should be strict > than 0

Reply:
Thank you for your feedback.